app/views/customers/show.html.erb

```diff
@@ -15,7 +15,7 @@
           <tbody>
             <tr>
               <td><%= cr.service_category.name%></td>
-              <td><%= cr.expires_date.strftime("%B %e, %Y") %></td>
+              <td><%= cr.try(:expires_date).try(:strftime, "%B %e, %Y") %></td>
```

Comment:
we should solve this on the controller because were adding logic to protect sad path instead of addressing WHY we hit the sad path